### ISSUE TITLE
Feedback: Multi-Day booking for property rental

### ISSUE THREAD

**Issue by PeerRich** [calcom/cal.com#16026]
got this feedback from a customer looking to use cal.com with multi-day booking as a PMS (property management system)

while i dont think this is something we can ship anytime soon (lots of unknown unknowns) this could be something we can branch out into with multi-day bookings (think car rental, property rental, coworking space rental and any other generic service that sells multiple days vs hours)

> I hope you’re well. My name is XXX, and I’m an Airbnb host who values [Cal.com](http://cal.com/)’s scheduling and calendar management tools. I’d like to propose that [Cal.com](http://cal.com/) develop a Property Management System (PMS) tailored for short-term rental hosts.
> 
> Managing multiple listings across various OTAs is a significant challenge. A PMS integrated with [Cal.com](http://cal.com/) could streamline this process, providing a unified platform for hosts to manage bookings and avoid double bookings.
> 
> Looking ahead, the value of combined calendars will only increase. As awareness grows about time management issues—such as the productivity pitfalls of doom scrolling and the fragmentation of personal and professional time across multiple platforms—there will be a stronger demand for integrated solutions. A consolidated calendar system can help individuals and businesses manage their time more effectively, minimizing the cognitive load of juggling multiple scheduling tools and reducing the risk of time traps.
> 
> Freelancers are already managing their presence on channels like Fiverr and Upwork, while many people promote their services and schedule engagements on social media. The future will likely see more integrations as individuals and businesses navigate an increasing number of online platforms. A PMS that integrates multiple calendars into a single, cohesive view can provide immense value by enhancing efficiency and preventing scheduling conflicts.
> 
> Developing such a system now could position [Cal.com](http://cal.com/) at the forefront of this evolving market, setting a standard for how businesses and individuals manage their time across diverse channels.
> 
> I’d be happy to discuss this idea further. Thank you for considering it.
> 
> Best regards,

**Comment by PeerRich** [comment]
More context:

> Thank you for considering the idea of multi-day bookings for various services. It's great to know that you're open to exploring new functionalities like this.
> 
> I wanted to elaborate on what I thought would be helpful for us. 
> 
> The first is a combined calendar created using iCal and APIs, integrating bookings from multiple channels. Leading PMS not only unify calendars but also provide a unified inbox and rates whether at dynamic or not they take into consideration the overheads each channel charges to balance customer facing prices. Highly developed PMS's like hostaway have all these features. I would like to see [cal.com](http://cal.com/) do something similar but not just for hosts but other booked services.
> 
> As well as all of the features I've mentioned above I think one additional thing there would be helpful and particularly a challenge that I face with is personalising calendars for each user group. In the example below I will explain that as a host our cleaners would benefit for an additional calendar dedicated to staff tasks, automatically generated based on trigger events and the booking calendar. For example, in a hosting company context, a turnover cleaning task could be scheduled between 11 am and 1 pm on the final day of each booking, helping cleaners manage their workload and ensure tasks are completed on time. This could be provided and a cleaning rust calendar. Similarly, for car rentals, tasks like vehicle inspections, cleanings, or refueling could be automatically scheduled based on the booking schedule.
> 
> Integrating a Property Management System (PMS) into [Cal.com](http://cal.com/) could be a significant enhancement. Key elements of a PMS—such as providing a combined calendar, messaging, and managing rates—are essential for any type of booking system. These features streamline operations, reduce the risk of double bookings, and facilitate efficient communication with clients and staff.
> 
> Although this might not make it onto the roadmap this year, I believe that exploring the acquisition of a small company like Rentalwise could be a strategic move for [Cal.com](http://cal.com/). Rentalwise offers specific capabilities in the PMS space that could complement and accelerate the development of these features within [Cal.com](http://cal.com/).
> 
> Thank you again for considering these ideas. I'm excited about the potential for future developments and look forward to contributing further to any discussions on this topic.
> 
> Best regards,

**Comment by jlj-dev** [comment]
I agree that this functionality would be incredibly useful.  I schedule multi-day court trials that must be set only on certain days and the cal.com availability rules are perfect for picking a date on which such a trial can start. But then I can't schedule it to last 2, 3, or 5 days which is what I need to do to prevent it from offering those future days as 'available' when they are not.

**Comment by carstensbix** [comment]
Hi there, any updates on this? Could it potentially be included in version 5.0? I believe many people would be eager to contribute to its development, and it could open cal.com to an entirely new audience as well :)

**Comment by Makxzod** [comment]
Hi there, any updates on this? Could it potentially be included in version 5.0? I believe many people would be eager to contribute to its development, and it could open cal.com to an entirely new audience as well.

**Comment by Makxzod** [comment]
@PeerRich Hi there, any updates on this? Could it potentially be included in version 5.0? I believe many people would be eager to contribute to its development, and it could open cal.com to an entirely new audience as well.

**Comment by PeerRich** [comment]
continuing here #20441
